.typos.toml

```diff
@@ -0,0 +1,10 @@
+# If there are words that you think should not be treated as typo.
+# Please list here along with a comment.
+[default.extend-words]
+"MUTLI" = "MUTLI"
+"SPLITTED" = "SPLITTED"
+"ba" = "ba"
+"hel" = "hel"
```

Comment:
are these needed?

Reply:
same words in tests seems intend, not a typo, so just ignore these ones, make the check happy